flink-core/src/test/java/org/apache/flink/api/common/ExecutionConfigTest.java

```diff
@@ -192,7 +189,7 @@ public void testReadingDefaultConfig() {
         // mutate config according to configuration
         executionConfig.configure(configuration, ExecutionConfigTest.class.getClassLoader());
 
-        assertThat(executionConfig, equalTo(new ExecutionConfig()));
+        assertEquals(executionConfig, new ExecutionConfig());
```

Comment:
According to Flink code style (https://flink.apache.org/contributing/code-style-and-quality-common.html#tooling), `assertThat` of `assertJ` is recommended.